.github/dependabot.yaml

```diff
@@ -7,6 +7,11 @@ updates:
       include: scope
       prefix: chore
     directory: /
+    ignore:
+      # keep everything compatible with older NodeJS runtimes
```

Comment:
Suggested edit: 
`Ignore >2x versions of Tempy as long as ESM is not supported`